src/utils/webrtc/simplewebrtc/localmedia.js

```diff
@@ -1,26 +1,24 @@
-/* global module */
+import util from 'util'
 
-const util = require('util')
-
-const mockconsole = require('mockconsole')
-const WildEmitter = require('wildemitter')
+import mockconsole from 'mockconsole'
+import WildEmitter from 'wildemitter'
 
 // Only mediaDevicesManager is used, but it can not be assigned here due to not
 // being initialized yet.
-const BlackVideoEnforcer = require('../../media/pipeline/BlackVideoEnforcer.js').default
-const MediaDevicesSource = require('../../media/pipeline/MediaDevicesSource.js').default
-const SpeakingMonitor = require('../../media/pipeline/SpeakingMonitor.js').default
-const TrackConstrainer = require('../../media/pipeline/TrackConstrainer.js').default
-const TrackEnabler = require('../../media/pipeline/TrackEnabler.js').default
-const TrackToStream = require('../../media/pipeline/TrackToStream.js').default
-const VirtualBackground = require('../../media/pipeline/VirtualBackground.js').default
-const webrtcIndex = require('../index.js')
-const getScreenMedia = require('./getscreenmedia.js')
+import BlackVideoEnforcer from '../../media/pipeline/BlackVideoEnforcer.js'
+import MediaDevicesSource from '../../media/pipeline/MediaDevicesSource.js'
+import SpeakingMonitor from '../../media/pipeline/SpeakingMonitor.js'
+import TrackConstrainer from '../../media/pipeline/TrackConstrainer.js'
+import TrackEnabler from '../../media/pipeline/TrackEnabler.js'
+import TrackToStream from '../../media/pipeline/TrackToStream.js'
+import VirtualBackground from '../../media/pipeline/VirtualBackground.js'
+import * as webrtcIndex from '../index.js'
```

Comment:
When `require('../index.js')` was used all the exported elements had to be accessed through the `webrtcIndex` object. However, only `mediaDevicesManager` is used in the whole file, so now it would be possible to import just that variable and change the use of `webrtcIndex.mediaDevicesManager` to `mediaDevicesManager`.